docs/docs/configuration/filtering.md

```diff
@@ -483,39 +478,15 @@ More info about the helper functions are in the library [here](https://github.co
 
 You can find more example policies [here](https://github.com/aquasecurity/trivy/tree/{{ git.tag }}/pkg/result/module.go)
 
-## By Inline Comments
-
+### By Vulnerability Exploitability Exchange (VEX)
 |     Scanner      | Supported |
 |:----------------:|:---------:|
-|  Vulnerability   |           |
-| Misconfiguration |     ✓     |
+|  Vulnerability   |     ✓     |
+| Misconfiguration |           |
```

Comment:
OpenVex and csaf support misconfigs, secrets and licenses.
We can do something like this:
```suggestion
|  Vulnerability   |     ✓     |
| Misconfiguration |     ✓ [^2]   |

...
[^2]: Only for `OpenVEX` and `CSAF`.
```

Reply:
Can you elaborate on it? VEX is for vulnerabilities as it stands for Vulnerability Exploitability Exchange.

Reply:
oh... Sorry.
I confused myself. Don't pay attention. Everything is fine here.